Formula/serf.rb

```diff
@@ -18,87 +18,18 @@ class Serf < Formula
   end
 
   depends_on "go" => :build
+  depends_on "govendor" => :build
```

Comment:
I'll double check, but given how (not so) often people update resources hardcoded into formulae I suspect it's not in sync.